pkg/workload/resources.go

```diff
@@ -125,17 +129,16 @@ func AdjustResources(ctx context.Context, cl client.Client, wl *kueue.Workload)
 // ValidateResources validates that requested resources are less or equal
 // to limits.
 func ValidateResources(wi *Info) error {
-	podsetsPath := field.NewPath("podSets")
 	// requests should be less than limits.
 	var allReasons []string
 	for i := range wi.Obj.Spec.PodSets {
 		ps := &wi.Obj.Spec.PodSets[i]
-		psPath := podsetsPath.Child(ps.Name)
+		podSpecPath := podSetsPath.Index(i).Child(ps.Name).Child("template").Child("spec")
```

Comment:
Is this correct though? Iirc there is no child `ps.Name`, just index, because pods sets are represented as a list, the name is only on a field.

Reply:
That sounds reasonable.

Reply:
I trimmed the PodSet name from pahts.